secure-releases.md

```diff
@@ -0,0 +1,60 @@
+# Secure Releases and CVE Management for the JS Ecosystem
+
+## Secure Releases
+
+In order for your JS software project to be safe, secure, and reliable, the versions it releases need to follow security best practices. Below are the OpenJS Foundations recommended best practices for secure releases.
+
+### Two-factor Authentication
+
+Two-factor authentication (2FA) means having something more than just your password to confirm you're you. The second factor can be a one-time code using an app like Google Authenticator, a hardware key (such as Yubikey or Titan), or a passkey using touch ID, face ID, or something else that's stored locally on your device. Note: do not *ever* use SMS for 2FA unless it's the only mechanism offered, since SMS is vulnerable to SIM swap and number forgery attacks.
```

Comment:
"something else" is vague. consider eliminating.. ambiguity can be problematic when discussing security.